packages/react-jss/src/types/index.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import type {
+  InternalStyleSheetOptions,
+  Rule,
+  ToCssOptions,
+  RuleOptions,
+  JssStyle
+} from 'jss/lib/types'
```

Comment:
All of those should be imported from `jss`

Reply:
Fixed